.env.example

```diff
@@ -24,6 +24,7 @@ AUTO_WITHDRAW_PAYMENTS="true" #should payments be automatically withdrawn after
 AUTO_WITHDRAW_REFUNDS="true" #should refunds be automatically withdrawn. If not set to false. Defaults to true
 
 #The following data is only needed while seeding
+DEFAULT_ADMIN_KEY="DefaultAdminKey"  #This is the Default Admin Key.
```

Comment:
I think there was something unclear in the description, do not add a default admin key here, just use one in the seeding script something like "DefaultUnsecureAdminKey", it makes no sense to have it configurable in the .env just use it if the Admin key is not set